src/main/java/com/github/checkstyle/generatepatchfile/GeneratePatchFile.java

```diff
@@ -259,4 +322,17 @@ public void generatePatch(int runPatchNum) throws Exception {
         }
         generateSummaryIndexHtml();
     }
+
+    /**
+     * To generate patch file through git command.
+     * @param runPatchNum num of patch file between two commits
+     * @throws Exception exception
+     */
+    public void generatePatchWithGitCommand(int runPatchNum) throws Exception {
+        for (int i = 1; i < runPatchNum; i++) {
+            generateDiffPatchWithGitCommand("" + i);
+            checkoutWithGitCommand();
```

Comment:
Isn't it easier to execute `"git format-patch -" + runPatchNum` and then rename patch files? This doesn't require doing checkout every time and repository will not be switched to another revision after generation.

Reply:
done